### DISCUSSION TITLE
Iris 3315-L (Leak Detector)  Any chance of a driver getting added?

### DISCUSSION THREAD

**Discussion by hobbyboy** [Koenkk/zigbee2mqtt#11987] [Devices]
Hi, moving from Hubitat to HA, and I've got 10 of these Iris leak detectors going.  Any chance a driver can be written for zigbee2mqtt?

I'll see this in the logs after pairing and a 'wet' message gets sent:
Received message from unsupported device with Zigbee model '3315-L' and manufacturer name 'CentraLite'

Thanks

**Comment by ogiewon** [comment]
I would like to see the Lowes Iris 3315-L leak sensor supported as well.  Centralite was the OEM for these sensors.

**Comment by ogiewon** [comment]
@hobbyboy - Just an FYI - I just submitted a pair of Pull Requests to add the Iris 3315-L to Zigbee2MQTT.  
https://github.com/Koenkk/zigbee-herdsman-converters/pull/4120
https://github.com/Koenkk/zigbee2mqtt.io/pull/1328

Hopefully this change will make it into an upcoming release.

**Reply by Koenkk** [reply]
Will be included in the next release (1 May) and in the latest dev branch in a few hours from now (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
